gtest/src/manager.rs

```diff
@@ -304,6 +305,32 @@ impl ExtManager {
         }
     }
 
+    pub(crate) fn is_user(&self, id: &ProgramId) -> bool {
+        !self.actors.contains_key(id) || matches!(self.actors.get(id), Some((Actor::User, _)))
+    }
+
+    pub(crate) fn add_value_to(&mut self, id: &ProgramId, value: Balance) {
+        let (_, balance) = self.actors.entry(*id).or_insert((Actor::User, 0));
+        *balance = balance.saturating_add(value);
+    }
+
+    pub(crate) fn actor_balance(&self, id: &ProgramId) -> Option<Balance> {
+        self.actors.get(id).map(|(_, balance)| *balance)
+    }
+
+    pub(crate) fn claim_value_from_mailbox(&mut self, id: &ProgramId) {
```

Comment:
Don't you want to provide different api for claiming value from concrete messages in mailbox, not from the all in the same moment?

Reply:
Added `Mailbox::claim_value` function (https://github.com/gear-tech/gear/pull/1125/commits/7a91463fbb8dfa1c726d1ac88601641d948498d5).